matrix-appservice-irc/irc-registration.yaml

```diff
@@ -0,0 +1,20 @@
+id: 8932b3435f0dbb697307801fbcaedad2da2d29fa71c30d70ef433a21c0585db6
+hs_token: 711663d558bd4be0d41976a89347e86b14f7addcb15b661b14101e1f83196a53
+as_token: d81e0ccf5435cca28ee0140ccef48cf935810f95836865dc7014353c15e4a0fa
```

Comment:
What do these mean? Are they secret? If they are secret please remove them, otherwise write a comment explaining what they mean.

Reply:
pretty sure they are secret

Reply:
Yeah, these are secrets, but these can be easily rotated out once they're moved out. This is the only location where the tokens exist